CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#4251](https://github.com/open-telemetry/opentelemetry-python/pull/4251))
 - Fix recursion error with sdk disabled and handler added to root logger
   ([#4259](https://github.com/open-telemetry/opentelemetry-python/pull/4259))
+- sdk: setup EventLogger when OTEL_PYTHON_LOGGING_AUTO_INSTRUMENTATION_ENABLED is set
```

Comment:
@lzchen do I need to fix this to be under a new "Unreleased" heading?

Reply:
https://github.com/open-telemetry/opentelemetry-python/pull/4282#discussion_r1842928114

Reply:
Fixed